kartothek/io/testing/build_cube.py

```diff
@@ -269,6 +273,52 @@ def test_parquet(driver, function_store):
         pdt.assert_frame_equal(df_actual.reset_index(drop=True), df_expected)
 
 
+@pytest.mark.filterwarnings("ignore::UnicodeWarning")
+@pytest.mark.parametrize("chunk_size", [None, 1, 2, 3])
```

Comment:
`[None, >0]` should be sufficient. No need to have multiple integers in here

Reply:
Note that these do test different edge cases (e.g. whether number of rows is divisible by chunk size). But I agree this is probably the wrong abstraction level to test this.

Reply:
indeed, the test makes sense but not on cube level. the only thing we actually want to test on cube level is "did I actually use the provided serializer or not".

You could even take this a level further by implementing a mock serializer and we merely verify if it was called. This way we'd not do any IO in the test. After all, we rely on the serializer to be implemented correctly so we don't need to test all peculiarities of it. We might finally have fast tests this way :D

Note: Completely fine if you don't go down that route but eventually I would hope we can test kartothek on this level instead of always performing full end-to-end tests